src/main/java/de/upb/crypto/clarc/incentive/CreditInstance.java

```diff
@@ -0,0 +1,59 @@
+package de.upb.crypto.clarc.incentive;
+
+import de.upb.crypto.clarc.protocols.arguments.SigmaProtocol;
+import de.upb.crypto.clarc.protocols.parameters.Announcement;
+import de.upb.crypto.clarc.protocols.parameters.Challenge;
+import de.upb.crypto.clarc.protocols.parameters.Response;
+import de.upb.crypto.craco.sig.ps.PSExtendedVerificationKey;
+import de.upb.crypto.craco.sig.ps.PSSignature;
+import de.upb.crypto.craco.sig.ps.PSSigningKey;
+import de.upb.crypto.math.interfaces.structures.GroupElement;
+import de.upb.crypto.math.structures.zn.Zp;
+
```

Comment:
JavaDoc missing

Reply:
Just a short sentence for each class, maybe?

Reply:
I add that in a minute.